src/gui/gui.zig

```diff
@@ -376,6 +377,14 @@ pub fn closeWindow(id: []const u8) void {
 	std.log.warn("Could not find window with id {s}.", .{id});
 }
 
+pub fn showNotification(message: []const u8) void {
+	notification.onOpen();
+	openWindow("notification");
```

Comment:
If the window is already open, we need to close it first, as there can only be one instance of a given window.